custom_tesseract_training/README.md

```diff
@@ -22,11 +22,11 @@
 (all commands done in the terminal, with Docker open, in your base environment (you will connect to a virtual environment in docker)
 
 * Run the Dockerfile using this command from the repo root. 
-  * docker-compose -f docker.dev.yml up -d
+  * docker-compose -f docker.dev.yml up -d train
```

Comment:
this should have a --build flag on it so that when the dev fires up the container it uses the newest version of the code. Without that flag, if a previous build exists docker will just use that since its cached. That means that the if the user changes code and reruns this command, nothing will change. So ideallyyou want 

`docker-compose -f docker.dev.yml up --build -d train`

☝️ but I also recommend renaming `docker.dev.yml` to `docker-compose.yml` as that is more standardized practice

Reply:
Hey @nlittlepoole I Updated the commands and renamed the file.